### PR TITLE
[orc8r][nprobe] Add storage interface for nprobe data

### LINKED ISSUE #4730: [X2][orc8r] Support a stateless implementation using subscriber state

### PR DESCRIPTION
**Pull request by ymasmoudi**
Signed-off-by: YOUSSEF EL MASMOUDI <ymasmoudi@fb.com>

## Summary

NProbe service will provide Lawful Inspection as described in the design quip "Lawful Interception in Magma"
This PR implements a storage interface used by the service to get/store/delete nprobe data from the blobstore.

The nprobe data will include the following information for now,
1/ Timestamp of last record exported
2/ TargetID (IMSI/IMEI/MSISDN)
3/ Sequence Number for the current nprobe task

Nprobe data is deleted when a task is deleted through swagger.

## Test Plan
```
ymasmoudi-mbp:storage ymasmoudi$ go test ./...  -v
=== RUN   TestStoreNProbeData
--- PASS: TestStoreNProbeData (0.00s)
    storage_blobstore_test.go:60: PASS:	StartTransaction(string)
    storage_blobstore_test.go:61: PASS:	Rollback()
    storage_blobstore_test.go:61: PASS:	CreateOrUpdate(string,blobstore.Blobs)
    storage_blobstore_test.go:61: PASS:	Commit()
    storage_blobstore_test.go:77: PASS:	StartTransaction(string)
    storage_blobstore_test.go:78: PASS:	Rollback()
    storage_blobstore_test.go:78: PASS:	Get(string,storage.TypeAndKey)
    storage_blobstore_test.go:78: PASS:	Commit()
    storage_blobstore_test.go:92: PASS:	StartTransaction(string)
    storage_blobstore_test.go:93: PASS:	Rollback()
    storage_blobstore_test.go:93: PASS:	Delete(string,[]storage.TypeAndKey)
    storage_blobstore_test.go:93: PASS:	Commit()
PASS
ok  	magma/lte/cloud/go/services/nprobe/storage	0.632s
```
Nprobe_manager changed to get/update nprobe data and tested on a local setup.

### CODE REVIEW

**Review by hcgatewood — COMMENTED** [review]
Kicking to @mpgermano

File: lte/cloud/go/services/nprobe/obsidian/models/swagger.v1.yml [hunk 1]
```diff
@@ -292,3 +292,20 @@ definitions:
         type: string
         x-nullable: false
         example: '127.0.0.1:4040'
+
+  network_probe_data:
```
Comment (mpgermano) [inline lte/cloud/go/services/nprobe/obsidian/models/swagger.v1.yml:296]: Should this definition have any required attributes?

**Review by mpgermano — APPROVED** [review]
LGTM. Please address the `required` attributes if needed.

File: lte/cloud/go/services/nprobe/obsidian/models/swagger.v1.yml (hunk 1 above)
Comment (ymasmoudi) [reply lte/cloud/go/services/nprobe/obsidian/models/swagger.v1.yml:296]: fixed. could you help merge this pr please ?

**Pull request merged**